testsuite/databases/redis/service.py

```diff
@@ -31,6 +35,7 @@ class ServiceSettings(typing.NamedTuple):
     master_ports: typing.Tuple[int, ...]
     sentinel_port: int
     slave_ports: typing.Tuple[int, ...]
+    cluster_ports: typing.Tuple[int, ...]
```

Comment:
do sentinel redis and cluster redis share same fields? If they don't please split into two classes